packager/live_packager_export.cc

```diff
@@ -104,13 +106,13 @@ LivePackagerStatus_t livepackager_package_timedtext_init(
   shaka::SegmentData input_seg(seg, seg_len);
   shaka::FullSegmentBuffer out;
   shaka::Status status = lp->inner->PackageTimedText(input_seg, out);
-  std::string status_message = status.ToString();
   if (!status.ok()) {
+    std::string status_message = status.ToString();
     return LivePackagerStatus_s{strdup(status_message.c_str()), status.ok()};
```

Comment:
Need to do the same here for `strdup`

Reply:
But we are already in the scope of `!status.ok()`.

Reply:
Oops - I missed that. 
Nit: For all of these, we don't need to create a temporary string and can call `status.ToString().c_str()` instead.